src/components/Movies.jsx

```diff
@@ -0,0 +1,22 @@
+import Movie from './Movie'
+import '../styles/movies.scss'
+
+const Movies = ({ movies, viewTrailer, closeCard }) => {
+
+    return (
+        <div data-testid="movies">
+            {movies.movies.results?.map((movie) => {
```

Comment:
If we need here only movies array I think it will be great to destructure it from slice on a parent level and it will be just as:
`movies.results?.map((movie) => {`